css-scroll-snap-1/Proposal.html

```diff
@@ -1584,8 +1584,8 @@ <h2 class="no-num no-ref heading settled" id="property-index"><span class="conte
       <th scope="col">Inh.
       <th scope="col">%ages
       <th scope="col">Media
-      <th scope="col">Ani­mat­able
-      <th scope="col">Com­puted value
```

Comment:
What changed in these lines?

Reply:
Same zero-width character splitting the word